### PR TITLE
Configuration file changes

### PR DESCRIPTION
**Pull request by MetRonnie**
Sister PR to https://github.com/cylc/cylc-flow/pull/3802 (although there isn't really any change in this, most of the doc updates were in that one)

### CODE REVIEW

File: src/suite-design-guide/general-principles.rst [hunk 1]
```diff
@@ -433,13 +433,13 @@ Sharing by inheritance of task families is recommended when more than a few
 configuration items are involved.
 
 The simplest application of inheritance is to set global defaults in the
-``[[runtime]][root]`` namespace that is inherited by all tasks.
+``[runtime][root]`` namespace that is inherited by all tasks.
```
Comment (datamel) [inline src/suite-design-guide/general-principles.rst:436]: Should this be 
```suggestion
``[runtime][[root]]`` namespace that is inherited by all tasks.
```

**Review by datamel — COMMENTED** [review]
I have checked out your branch and built the docs. It looks grand. Just wondering about the [runtime][root],   this is currently [runtime] [[root]], no?

File: src/suite-design-guide/general-principles.rst (hunk 1 above)
Comment (MetRonnie) [reply src/suite-design-guide/general-principles.rst:436]: I thought when writing a section/setting out "flat" we're supposed to ignore the multiple brackets?

Comment (MetRonnie) [reply src/suite-design-guide/general-principles.rst:436]: https://cylc.github.io/cylc-doc/8.0a2/html/tutorial/scheduling/graphing.html#shorthand

Comment (datamel) [reply src/suite-design-guide/general-principles.rst:436]: Ah pass, I've not come across that yet but it is entirely possible.

Comment (MetRonnie) [reply src/suite-design-guide/general-principles.rst:436]: Yes, if you look at e.g. https://cylc.github.io/cylc-doc/8.0a2/html/reference/config/suite.html#suite.rc[meta]URL it's also how you write rst references
```rst
:cylc:conf:`flow.cylc[runtime][<namespace>][meta]URL`
```

**Review by datamel — APPROVED** [review]

**Comment by MetRonnie** [comment]
1 reviewer should be fine

**Pull request merged**